smart-contracts/deployments/default/BridgeRegistry.json

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
Are you also going to remove all the files from git in `smart-contracts/deployments/default/*` @ElliotFriedman ?

Reply:
yes, I will remove those.